sdk/examples/xo_python/sawtooth_xo/xo_client.py

```diff
@@ -179,34 +179,22 @@ def _send_xo_txn(self, name, action, space="", wait=None,
         if wait and wait > 0:
             wait_time = 0
             start_time = time.time()
-            self._send_request(
+            response = self._send_request(
                 "batches", batch_list.SerializeToString(),
                 'application/octet-stream',
                 auth_user=auth_user,
                 auth_password=auth_password
             )
             while wait_time < wait:
-                status = self._get_status(
+                self._get_status(
```

Comment:
Without checking the status, this will just keep looping until the wait time expires.